src/types/collection.graphql

```diff
@@ -55,6 +55,8 @@ type Collection {
   hasFormats: Boolean
   "True if there are granules associated with the collection."
   hasGranules: Boolean
+  "True if there is an OPeNDAP service RelatedURL associated with the collection."
+  hasOpendap: Boolean
```

Comment:
Is `hasOpendap` a field returned by CMR? If not than we don't want to provide it as a returnable value here

Reply:
https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html#c-has-opendap-url
curl "https://cmr.earthdata.nasa.gov/search/collections?has_opendap_url=true"

Reply:
It is not a returnable field, it is filtering CMR results by relatedUrls so it will always be null. It should still be included in the collectionsInput though

`query ($params: CollectionsInput) {
  collections(params: $params) {
    items {
      conceptId
    }
  }
}`

with:
```
{
  "params":{
    "hasOpendapUrl":true
  }
}
```